x/accounts/go.mod

```diff
@@ -175,4 +175,5 @@ replace (
 	cosmossdk.io/x/mint => ../mint
 	cosmossdk.io/x/slashing => ../slashing
 	cosmossdk.io/x/staking => ../staking
+	cosmossdk.io/collections => ../../collections
```

Comment:
Lets avoid to add replace everywhere, can you revert them all? We can see in a follow up if the implemented time key in collection is and should be state compatible with the one we had in the sdk.